src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/tests/__snapshots__/DatePicker.test.js.snap

```diff
@@ -1,1109 +1,1270 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`DatePicker should render 1`] = `
-<DatePicker
-  onChange={[MockFunction]}
-  options={Object {}}
-  valid={true}
-  value={null}
->
+exports[`DatePicker should not open overlay on icon-click when disabled 1`] = `
+<div>
   <div>
-    <div />
-    <Popover
-      anchorElement={
-        <div>
-          <label
-            class="input default left"
-          >
-            <div
-              class="prependedContainer default"
-            >
-              <span
-                aria-label="su-calendar"
-                class="su-calendar clickable icon default iconClickable"
-                role="button"
-                tabindex="0"
-              />
-            </div>
-            <input
-              placeholder="MM/DD/YYYY"
-              type="text"
-              value=""
-            />
-          </label>
+    <label
+      class="input default left disabled"
+    >
+      <div
+        class="prependedContainer default"
+      >
+        <span
+          aria-label="su-calendar"
+          class="su-calendar icon default"
+        />
+      </div>
+      <input
+        disabled=""
+        placeholder="MM/DD/YYYY"
+        type="text"
+        value=""
+      />
+    </label>
+  </div>
+</div>
+`;
+
+exports[`DatePicker should not open overlay on icon-click when disabled 2`] = `
+"<div>
```

Comment:
Shouldn't that be empty now? Looks like the click opened the overlay. Also don't know if I would use snapshot testing here, since checking for the `open` attribute of the `Overlay` component would be enough.